src/common/resources/fiscalYear.js

```diff
@@ -0,0 +1,20 @@
+import { get } from 'lodash';
+
+import { baseManifest } from '@folio/stripes-acq-components';
+
+import { FISCAL_YEARS_API } from '../const';
+
+export const fiscalYearsResource = {
+  ...baseManifest,
+  path: FISCAL_YEARS_API,
+  records: 'fiscalYears',
+};
+
+export const fiscalYearByIdResource = {
+  ...baseManifest,
+  path: (queryParams, pathComponents, resourceData, logger, props) => {
+    const fiscalYearId = get(props, ['resources', 'budget', 'records', 0, 'fiscalYearId']);
```

Comment:
IMHO, this is not a common resource. I think it makes sense to put such specific path fns directly in the manifest. I doubt that it could be re-used, since it's tightly coupled with another manifest attribute. 
By `fiscalYearByIdResource` I would expect `path: '${FISCAL_YEARS_API}/!fiscalYearId'` - it gets `fiscalYearId` from component props

Reply:
Fixed